tests/lib/schema_validation_test.py

```diff
@@ -83,6 +84,28 @@
                 'next-hop-address': 'fe80::1'
             }
         ]
+    },
+    DNS: {
+        "config": {
+            "name-servers": [
```

Comment:
How about calling this just `servers`?

Reply:
Changed to [RFC 7317][1] suggested names: `dns-resolver`, `server`, `search`.

Example:
```yaml
-
dns-resolver:
  config:
    search:
    - example.com
    - example.org
    server:
    - 192.168.122.1
    - 2001::8
    - 8.8.8.8
  running:
    search:
    - example.com
    - example.org
    server:
    - 8.8.8.8
    - 2001::8
    - 192.168.122.1
```

[1]: https://tools.ietf.org/html/rfc7317#section-3.3